src/rating-indicator.scss

```diff
@@ -0,0 +1,220 @@
+    margin-right: $fd-rating-indicator-default-spacing-between-icons;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    @include fd-rtl() {
+      float: right;
+    }
+
+    &:hover::before {
+      opacity: var(--sapRating_Indicator_Selected_Hover_Opacity);
+      transition: opacity $fd-animation-speed ease-in;
+    }
+  }
+
+  &__dynamic-text {
+    @include fd-reset();
+    @include fd-ellipsis();
+
+    font-size: var(--sapFontSmallSize);
+    color: var(--sapContent_LabelColor);
+    margin-left: 0.5rem;
+  }
+
+  &__hide-dynamic-text {
+    > .#{$block}__dynamic-text {
+      display: none;
+    }
+  }
+
+  @include fd-disabled() {
+    pointer-events: none;
+    opacity: var(--sapRating_Indicator_Disabled_Control_Opacity);
+
+    .#{$block}__input {
+      @include fd-icon-glyph('favorite') {
+        color: var(--sapRating_Indicator_Disabled_Selected_Color);
+      }
+
+      &:checked ~ input + .#{$block}__label {
+        @include fd-icon-glyph('favorite') {
+          color: var(--sapRating_Indicator_Disabled_Unselected_Color);
+        }
+      }
+    }
+  }
+
+  @include fd-readonly() {
+    pointer-events: none;
+
+    .#{$block}__input {
+      @include fd-icon-glyph('favorite') {
+        color: var(--sapRating_Indicator_Readonly_Selected_Color);
+      }
+
+      &:checked ~ input + .#{$block}__label {
+        @include fd-icon-glyph('favorite') {
+          color: var(--sapRating_Indicator_Readonly_Unselected_Color);
+        }
+      }
+    }
+  }
+
+  @include fd-flex-center();
+
+  &.is-display-mode {
+    pointer-events: none;
+
+    .#{$block}__label {
+      @include fd-icon-glyph('favorite') {
+        color: var(--sapRating_Indicator_Display_Only_Selected_Color);
+      }
+
+      &::before {
+        font-size: 1rem;
+      }
+
+      margin-right: 0.125rem;
+    }
+
+    .#{$block}__input {
+      &:checked ~ input + .#{$block}__label {
+        @include fd-icon-glyph('favorite') {
+          color: var(--sapRating_Indicator_Display_Only_Unselected_Color);
+        }
+      }
+    }
+  }
+
+  &--half-star {
+    @include fd-rating-indicator-with-half-star($fd-rating-indicator-default-font-size, $fd-rating-indicator-default-spacing-between-icons);
+  }
+
+  // BLOCK MODIFIERS ************
+  // sizes
+  @each $set-name, $size-set in $fd-rating-indicator-sizes {
+    $fd-rating-indicator-sizes-font-size: map_get($size-set, "fontSize");
+    $fd-rating-indicator-sizes-spacing-between-icons: map_get($size-set, "spacingBetweenIcons");
+    &--#{$set-name} {
+      @include fd-reset-spacing();
+      $controlMargin: map_get($size-set, "controlMargin");
+
+      @if $controlMargin {
+        margin: $controlMargin;
+      }
+
+      .#{$block}__input {
+        width: $fd-rating-indicator-sizes-font-size;
+        height: $fd-rating-indicator-sizes-font-size;
+      }
+
+      .#{$block}__label {
+        margin-right: $fd-rating-indicator-sizes-spacing-between-icons;
```

Comment:
Margin doesn't support RTL 
![image](https://user-images.githubusercontent.com/26483208/91840491-0e9e8f80-ec51-11ea-8b08-521bdb048d23.png)
You can change it to `margin-left`, or switch `&:last-child` with `&:first-child` on RTL

Reply:
fixed